lib/iso.rb

```diff
@@ -35,17 +40,27 @@ def execute options
     isoname = "#{hostname}.iso"
     work_dir = File.realdirpath(ENV['PWD'] + "/mkvm")
     tmp_dir = "#{work_dir}/#{hostname}"
-
-    # TODO: handle exceptions
     FileUtils.mkdir_p tmp_dir
 
-    # create the ISO template directory
-    FileUtils.cp_r srcdir, "#{tmp_dir}/isolinux"
+    # Fetch isolinux content.
+    if options[:isolinux_remote_host]
+      remote_path = "/var/satellite/rhn/kickstart/ks-rhel-x86_64-server-#{options[:major]}-#{options[:major]}.#{options[:minor]}/isolinux"
+
+      require 'net/scp'
+      Net::SCP.start(options[:isolinux_remote_host], ENV['USER']) do |scp|
+        scp.download! remote_path, tmp_dir, options = { :recursive => true }
```

Comment:
I don't find an api method for accessing these and the are not directly accessible via https.